stories/kitchen-sink.stories.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import KitchenSink from '../app/components/kitchen-sink';
+import common from './common'
+
+const testType = {
+	"_id": "56ce331e7957d17202e996d6",
+	"name": "Intro",
+	"harmony": [],
+	"id": "9okDr"
+}
+
+export default {
+    component: KitchenSink,
+    decorators: [
+        Story => {
+            common.outerSetup()
+            return <Story />
+        }
+    ]
+}
+
+export const KitchenSinkStory = () => {
```

Comment:
Since this is already part of KitchenSink - the component name, the name of this story should be more about what use case of KithenSink is being tested..  In this case I've seen Normal or Primary used. 

Also, can you please delete kitchen-sink.js since we won't need it after this is working.